asyncio/selector_events.py

```
@@ -39,6 +39,17 @@ def _test_selector_event(selector, fd, event):
         return bool(key.events & event)
 
 
+if hasattr(socket, 'TCP_NODELAY'):
+    def _set_nodelay(sock):
+        try:
+            sock.setsockopt(socket.IPPROTO_TCP, socket.TCP_NODELAY, 1)
+        except OSError:
+            pass
```

1. Hiding error is bad practice. Is any real case when it will fail?
2. Why not to embed that code into place where it is called ?